tests/dummy/app/routes/application.js

```diff
@@ -8,64 +7,6 @@ export default Ember.Route.extend({
 
   beforeModel (/* transition */) {
     return this.get('application').authenticate()
```

Comment:
trying to get fancy with the feature branch and TM support was causing a lot of issues. For some reason, Ember is messing with the URL and trying to redirect _before_ we get here (which is where I could control the redirect). This will just already return from auth at the root basepath.